extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -2218,7 +2218,11 @@ public static function onArticleCommentGetSquidURLs( $title, &$urls ) {
 	 * @return bool
 	 */
 	public static function onGetTalkPage( Title $title, Title &$talkPageTitle ) {
-		if ( !$title->isSubpage() && $title->getNamespace() == NS_USER ) {
+		global $wgEnableWallExt;
```

Comment:
I've never had a strong preference for that if the code isn't in a Nirvana controller, since `F::app()` is still a global ;), but switched since it's consistent with everywhere else in Wall. :)